gitops/snapshot.go

```diff
@@ -155,6 +158,9 @@ const (
 	// PipelineAsCodeSHAAnnotation is the commit which triggered the pipelinerun in build service.
 	PipelineAsCodeSHAAnnotation = PipelinesAsCodePrefix + "/sha"
 
+	// PipelineAsCodBranchAnnotation is the commit which triggered the pipelinerun in build service.
+	PipelineAsCodBranchAnnotation = PipelinesAsCodePrefix + "/branch"
```

Comment:
We have had this annotation `/branch` as target branch, see line 144 above

Reply:
nice catch ,
removed this duplication